.github/workflows/ci.yml

```diff
@@ -38,6 +38,9 @@ jobs:
       - name: Install tox
         run: |
           pip install --upgrade pip
+          git lfs install
+          git lfs fetch
+          git lfs checkout
```

Comment:
In other projects that use LFS in CI, I see the following changes:

```yml
    steps:
      - uses: actions/checkout@v2
        with:
          lfs: true
```

I think that is enough, and then you can omit these run steps.

Reply:
this change successfully converts the models from pointer files to their true content. Made this fix and the one comment below.